net/netxng/test/RRawFileNetXNG.cxx

```diff
@@ -48,6 +48,11 @@ TEST(RRawFileNetXNG, ReadV)
    options.fBlockSize = 0;
    auto f = std::make_unique<RRawFileNetXNG>("root://eospublic.cern.ch//eos/root-eos/testfiles/xrootd.test", options);
 
+   auto iovLimits = f->GetReadVLimits();
+   EXPECT_EQ(static_cast<std::uint64_t>(-1), iovLimits.fMaxTotalSize);
+   EXPECT_TRUE(iovLimits.HasReqsLimit());
```

Comment:
Possibly for a follow-up PR: it would be nice if there also was a test to check whether the default values or other values are correctly returned when they are expected. I don't know if there's a way to mock this behaviour?